src/libpgmoneta/prometheus.c

```diff
@@ -2377,6 +2392,27 @@ size_information(int client_fd)
    }
    data = pgmoneta_append(data, "\n");
 
+   // Append the WAL LSN of every server
+   data = pgmoneta_append(data, "#HELP pgmoneta_current_wal_lsn The current WAL log sequence number\n");
+   data = pgmoneta_append(data, "#TYPE pgmoneta_current_wal_lsn gauge\n");
+   for (int i = 0; i < config->number_of_servers; i++)
+   {
+      data = pgmoneta_append(data, "pgmoneta_current_wal_lsn{");
+
+      data = pgmoneta_append(data, "server=\"");
+      data = pgmoneta_append(data, config->servers[i].name);
+      data = pgmoneta_append(data, "\", ");
+
+      data = pgmoneta_append(data, "LSN=\"");
+      data = pgmoneta_append(data, config->servers[i].pgmoneta_current_wal_lsn);
```

Comment:
Consider the situation where servers[i].pgmoneta_current_wal_lsn is empty, because of server down or network error, you need to put "0/0" instead.

Reply:
I added the check in the new commit